docs/source/config_files.rst

```diff
@@ -1,9 +1,32 @@
 .. _config-files:
 
-Supported configuration files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Configuration Files
+~~~~~~~~~~~~~~~~~~~
 
-Below is a list of supported configuration files (roughly in the order of build priority).
+``repo2docker`` looks for configuration files in the repository being built
```

Comment:
...in the root directory of the selected repository to build.

..note::

The repo2docker buildpacks are inspired by Heroku's Build Packs.

Reply:
perhaps the heroku buildpack discussion be in `design.rst`?

Reply:
n.b. it is `binder/` not `.binder/` (leading dot).

Can we find a new word for composability? It seems to create the impression that you can combine anything with anything in terms of config but that doesn't work. Maybe "some of the configuration files can be combined to compose more complex setups?

Reply:
Good catch @betatim re leading dot.

Reply:
> some of the configuration files can be combined to compose more complex setups

I like this 👍

Reply:
@willingc I don't know if saying root first makes sense since `binder` takes precedent

Reply:
@betatim does the language for `binder` not clarify that it's not `.binder`?

Reply:
The .binder was only in a comment. Just wanted to make sure it didn't get copy&pasted in.

Maybe starting with "looks for configuration files in a top-level binder/ directory or in the root of the repository itself"?